ksqldb-functional-tests/src/test/resources/query-validation-tests/earliest-offset-udaf.json

```diff
@@ -24,20 +24,133 @@
       ]
     },
     {
-      "name": "earliest by offset with nulls",
+      "name": "null first - single - ignoring nulls",
+      "statements": [
+        "CREATE STREAM INPUT (ID BIGINT KEY, F0 INT) WITH (kafka_topic='test_topic', value_format='JSON');",
+        "CREATE TABLE OUTPUT AS SELECT ID, EARLIEST_BY_OFFSET(F0, true) AS L0 FROM INPUT GROUP BY ID;"
+      ],
+      "inputs": [
+        {"topic": "test_topic", "key": 0, "value": {"F0": null}},
+        {"topic": "test_topic", "key": 0, "value": {"F0": 13}}
+      ],
+      "outputs": [
+        {"topic": "OUTPUT", "key": 0, "value": {"L0": null}},
```

Comment:
I was confused about this test, but the null here is not the null of the first row, but a null because there is no a value to return, right?

Reply:
correct